### PR TITLE
chore: Add e2e test for uploading hero image

### PR DESCRIPTION
**Pull request by abbyoung**
# SC-1451

## Proposed changes

* Add feature-based e2e tests for uploading a hero image and viewing it on the portal

## Setup

In `ussf-portal`, run `yarn services:up -d`

Make sure the test passes. You can run the single test by running `yarn e2e:test playwright/feature/article-hero.spec.ts`

**NOTE:**  Tests that update the postgres database have problems if ran more than once. This is something we're actively debugging but haven't solved yet.  When testing locally, you need to restart the keystone-cms container in docker before re-running. 

**UPDATE TO NOTE:** We've swapped in `TRUNCATE` instead of `DROP` to help alleviate some of the db errors we were seeing.

### CODE REVIEW

File: e2e/playwright/feature/article-hero.spec.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    /* Navigate to the Articles page */
+    await Promise.all([
+      page.waitForNavigation(),
+      page.locator('h3:has-text("Articles")').click(),
+    ])
+
+    /** Create a new article *****
+
+      Category: ORBITBlog
+      Title: My Test Article
```
Comment (gidjin) [inline e2e/playwright/feature/article-hero.spec.ts:58]: nit-pick(non-blocking): Article title is now different than the comment.

File: e2e/playwright/feature/article-hero.spec.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    await page
+      .locator('[aria-label="Side Navigation"] >> text=Articles')
+      .click()
+    await expect(page).toHaveURL('http://localhost:3001/articles')
+
+    await expect(
+      page.locator('tr:has-text("A Test Article") td:nth-child(3)')
+    ).toHaveText('Draft')
+
+    /* Log out as CMS author */
```
Comment (gidjin) [inline e2e/playwright/feature/article-hero.spec.ts:97]: praise: thanks for adding comments, I think it's helpful.

File: e2e/playwright/feature/article-hero.spec.ts [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    ).toBeVisible()
+
+    /* Navigate to the Articles page */
+    await Promise.all([
+      page.waitForNavigation(),
+      page.locator('h3:has-text("Articles")').click(),
+    ])
+
+    /* Publish article */
+    await page.locator('a:has-text("A Test Article")').click()
```
Comment (gidjin) [inline e2e/playwright/feature/article-hero.spec.ts:122]: thought: This test is dependent on the other in this file. I wonder if we should flag that in some way. Thought I'm not opposed to having tests in the same file depend on each other.

**Review by gidjin — APPROVED** [review]
LGTM!

**Comment by abbyoung** [comment]
Changes included in PR #186

**Pull request closed without merging**